csg.js

```diff
@@ -5713,21 +5719,17 @@ for solid CAD anyway.
         var center = CSG.parseOptionAs2DVector(options, "center", [0, 0]);
         var radius = CSG.parseOptionAsFloat(options, "radius", 1);
         var resolution = CSG.parseOptionAsInt(options, "resolution", CSG.defaultResolution2D);
-        var sides = [];
+        var points = [];
         var prevvertex;
-        for (var i = 0; i <= resolution; i++) {
+        for (var i = 0; i < resolution; i++) {
```

Comment:
It's the comparison change was because this is POINTS now, not SIDES. Can you check again?